GNUmakefile

```diff
@@ -52,10 +52,18 @@ test-compile:
 	fi
 	go test -c $(TEST) $(TESTARGS)
 
-generate-docs:
+dependencies:
+	@echo "Download go.mod dependencies"
+	@go mod download
+
+install-tools: dependencies
+	@echo "Installing tools from tools/tools.go"
+	@cat tools/tools.go | grep _ | awk -F '"' '{print $$2}' | xargs -tI {} go install {}
```

Comment:
Out of interest, why have you chosen this route instead of a bare`go install`?

Reply:
To clarify: `go install` needs a repo path e.g. `github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs` and so I can’t do something like `go install tools/tools.go`.

This means I needed to parse whatever is in the `tools/tools.go` file and grab all the imports. 

Admittedly there’s only one import so yeah I could just import it manually via the Makefile like so `go install github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs` but I didn’t want to couple the Makefile to a dependency that may not exist in future, nor have to update it every time a new tool is added.

Reply:
Yup, apologies, my understanding of `go install` was incorrect :)